BUILD.bazel

```diff
@@ -820,16 +798,19 @@ pyx_library(
         "linkstatic": 1,
         # see https://github.com/tensorflow/tensorflow/blob/r2.1/tensorflow/lite/BUILD#L444
         "linkopts": select({
-            "//:macos": [
+            "@bazel_tools//src/conditions:darwin": [
                 "-Wl,-exported_symbols_list,$(location //:src/ray/ray_exported_symbols.lds)",
             ],
-            "//:windows": [],
+            "@bazel_tools//src/conditions:windows": [
+            ],
             "//conditions:default": [
                 "-Wl,--version-script,$(location //:src/ray/ray_version_script.lds)",
             ],
         }),
     },
-    copts = COPTS + if_linux_x86_64(["-fno-gnu-unique"]),
+    copts = COPTS + select({
+        "@bazel_tools//src/conditions:linux_x86_64": ["-fno-gnu-unique"],
```

Comment:
I'm seeing the following error on macOS. I think you need to add a default condition here.
```
/Users/travis/build/ray-project/ray/BUILD.bazel:785:1: Configurable attribute "copts" doesn't match this configuration (would a default condition help?).
Conditions checked:
 @bazel_tools//src/conditions:linux_x86_64
```

Reply:
Yup I do have that fix locally! I haven't pushed it to avoid clogging the builds for now. Will hopefully push it when I fix some other things as well.

Reply:
Yup I do have that fix locally! I haven't pushed it to avoid clogging the builds for now. Will hopefully push it when I fix some other things as well.